tikv/split_region.go

```diff
@@ -327,11 +327,9 @@ func (s *KVStore) CheckRegionInScattering(regionID uint64) (bool, error) {
 		}
 		if err != nil {
 			err = bo.Backoff(retry.BoRegionMiss, errors.New(err.Error()))
+			return true, err
```

Comment:
This change breaks original logic. Before the patch, It did not return when `Backoff` returns `nil`.

Reply:
Don't panic 😄  Thanks for you contribution.